<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Group</class>
 <widget class="QWidget" name="Group">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Группа 1</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Интервал прихода</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="arrivalIntCB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>от:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="arrMinSB">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="value">
           <double>3.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>до:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="arrMaxSB">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="value">
           <double>6.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Интервал обработки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="procIntCB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>от:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="procMinSB">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="value">
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>до:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="procMaxSB">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="value">
           <double>30.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
